bash/code_to_gast/bash_helpers.py

```diff
@@ -0,0 +1,17 @@
+
+def word_to_gast(node):
+    literal = node.word
+    if literal == 'true':
+        return {'type': 'bool', 'value': 1}
+    elif literal == 'false':
+        return {'type': 'bool', 'value': 0}
+    else:
+        try:
```

Comment:
Nit:
For better readability consider helper functions to avoid nested try and excepts. See https://stackoverflow.com/questions/1265665/how-can-i-check-if-a-string-represents-an-int-without-using-try-except

Reply:
I had that before, but right now we are counting floats and ints the same way so I think I need the try and except.

Reply:
I could put it in a helper function though